compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -4452,21 +4452,39 @@ static TR::Register *vminDoubleHelper(TR::Node *node, TR::CodeGenerator *cg)
 
    node->setRegister(resReg);
 
+   //Choose comparison/minimum instructions to use based on element type
+   TR::InstOpCode::Mnemonic cmpOp;
+   TR::InstOpCode::Mnemonic minOp;
+
+   switch(type)
+     {
+     case TR::Float:
+       cmpOp = TR::InstOpCode::xvcmpeqsp;
+       minOp = TR::InstOpCode::xvminsp;
+       break;
+     case TR::Double:
+       cmpOp = TR::InstOpCode::xvcmpeqdp;
+       minOp = TR::InstOpCode::xvmindp;
+       break;
+     default:
+       TR_ASSERT(false, "vminFPHelper cannot be called on vector type %s\n", type.toString()); return NULL;
+     }
+
    /*
    To match with java library behavior, vmin should return NaN if EITHER of the operands is NaN.
    However, since the VSX minimum instruction only returns NaN when BOTH operands are NaN, some extra steps are needed.
    */
 
    //Check the first operand for NaN elements, and convert corresponding elements in second operand to NaN
```

Comment:
not related to your changes, but i think this implementation (& max one as well) hasn't taken into considerations of mis-match between hardware instruction and java semantic about -0 and +0. the former treats them as the same, while the latter doesn't. maybe fix them in the future.

Reply:
I'll open a separate PR for this, so I think we can proceed with this PR as is for now